### DISCUSSION TITLE
part-ii/2021-89/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#362] [Question comments]
Part II 2021 Logic and Set Theory: Paper 3, Section II, 16G 

https://questions.tripos.org/part-ii/2021-89/

**Comment by artie2000** [comment]
<details>
<summary>(a)</summary>

All except last bit is bookwork.

Note $2^\kappa>\kappa$. By well-ordering principle, $2^{\aleph_\alpha}=\aleph_\beta$, with $\beta>\alpha$. Exponentiate once more to get the result.

</details>
<details>
<summary>(b)</summary>

All of the bookwork goes through in ZF.

For $g:\mathcal{P}y\to\mathcal{P}x$, map $S\to f^{-1}(S)$.

For $f: \mathcal{P}\left(\omega_{\alpha} \times \omega_{\alpha}\right) \rightarrow \omega_{\alpha+1}$, map $r\to\left.\begin{cases}\text{ord}(r)&\text{if }r\text{ is a well-ordering}\\0&\text{o/w}\end{cases}\right\}$.

</details>
